articles/synapse-analytics/sql-data-warehouse/design-guidance-for-replicated-tables.md

```diff
@@ -185,6 +185,29 @@ To trigger a rebuild, run the following statement on each table in the preceding
 SELECT TOP 1 * FROM [ReplicatedTable]
 ```
 
+> [!NOTE]
+> If you are planning to rebuild the statistics of the uncached replicated table, make sure to update the statistics before triggering the cache. Updating statistics will invalidate the cache, so the sequence is important.
+> 
+> Limitation: If you have a batch that contains the following sequence, the rebuild will not work in this case because of the update statistics. Instead start with UPDATE STATISTICS then at the end trigger the rebuild operation:
+> 
+> ```sql
+> -- Incorrect sequence. The rebuild operation will fail. Ensure that the rebuild operation is the last statement within the batch.
```

Comment:
`fail` implies an error message. Will the rebuild actually fail? I'm guessing no.

```suggestion
> -- Incorrect sequence. Ensure that the rebuild operation is the last statement within the batch.
```